ext/src/ruby_api/errors.rs

```diff
@@ -48,6 +49,17 @@ macro_rules! conversion_err {
     };
 }
 
+pub fn handle_wasm_error(store: &Store, error: anyhow::Error) -> Error {
```

Comment:
Any particular reason why this isn't part of the store?

Reply:
Wanted to try this out, but `rake` commands keep failing (also I didn't dig deep enough on the cause)

Reply:
> Any particular reason why this isn't part of the store?

Hmm, not really. I guess it can be! Once we merge #2 this will be StoreContextValue, but not a big deal: we can move the method.

Reply:
I was just thinking that it could be nicer, since the only thing that you'd need to pass in is the error.

Reply:
Done!